runtime/CSharp/runtime/CSharp/Antlr4.Runtime/ParserRuleContext.cs

```diff
@@ -252,7 +252,7 @@ public virtual ITerminalNode GetToken(int ttype, int i)
             return null;
         }
 
-#if NET45PLUS
+#if (NET45PLUS && !DOTNETCORE)
```

Comment:
I have no holistic understanding of both this **NET45PLUS** and **PORTABLE**. I believe it could be cleaned up somehow, but not in this PR.